scripts/linters/pylint_extensions_test.py

```diff
@@ -736,6 +736,21 @@ def test_space_after_docstring(self):
             self.checker_test_object.checker.visit_functiondef(
                 node_space_after_docstring)
 
+    def test_two_lines_empty_docstring_raise_correct_message(self):
+        node_with_docstring = astroid.extract_node(
+            u"""def func():
```

Comment:
We should also add a test for the case when the docstring is single line and empty (see below) and no error should be raised there (as it is a valid docstring).
```python
def func():
    """"""
    pass
```
The current test also ensures that we have solved the bug, and no (runtime) error is being raised, but for completeness we should add this test too, to verify **no** error message (of any kind) is raised for a correct docstring. (I leave it to you if you want to keep the current test or not)

Reply:
okay, Thanks for the info, I will skip writing a new test case for empty-docstring(feel free to mark these comments resolve).